SCARA_T7:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint_3
    pid: {p: 6000.0, i: 0.00, d: 40.0}